src/styles/utils/_variables.scss

```
@@ -0,0 +1 @@
+$-time-transition: 300ms;
```

Consider renaming the variable to a more meaningful values 
```suggestion
$transition-duration: 300ms;
```